synapse/lib/stormtypes.py

```diff
@@ -6987,3 +7029,21 @@ async def torepr(valu, usestr=False):
     if usestr:
         return str(valu)
     return repr(valu)
+
+async def tobuid(valu, noneok=False):
```

Comment:
Maybe I'm being too picky but seems like this should be `toiden` instead of `tobuid`

Reply:
Yea, i def debated it.  I was worried because we also use `iden` to describe/name `guid` vars.  I'll rename it `tobuidhex()` or something :)